### ISSUE TITLE
"Learning experiences become meaningless if they cannot be attributed to identifiable individuals and/or groups"

### ISSUE THREAD

**Issue by garemoko** [adlnet/xAPI-Spec#617]
This is a bit strong; anonymous data can be useful. Just needs re-wording a little. 

See http://tincanapi.com/2013/06/05/deep-dive-actor-agent/#comment-1836520796

**Comment by rklancer** [comment]
At [Concord Consortium](http://concord.org) we're experimenting with xAPI for use by educational researchers who might analyze user actions in tools like http://mw.concord.org/nextgen/#interactives and http://concord.org/projects/codap

In both cases, we expect to be able to glean insights from the actions (clickstreams, etc) of both logged-in users and anonymous users; not only is asking users to log in a barrier to casual use, but because much of our work is federally funded research, our ability to collect data from identifiable users is more restricted than it is for private businesses.

**Comment by garemoko** [comment]
Absolutely. I don't think this sentence is meant in any way to prevent use of anonymised agents; it's more intended to explain why identifiers can be useful and just a little badly worded.

**Comment by rklancer** [comment]
Good to see; thanks!

**Comment by canweriotnow** [comment]
Just as a note, going into 2.0 the whole IFI issue needs to be revisited with extreme prejudice as we start trying to build xAPI for IoT.

**Comment by garemoko** [comment]
I hit a use case recently (accounting professional bodies) where it would have been handy to have used an activity object as the actor of the statement. The work around is to use an account object with essentially a meaningless name, and focusing on the homePage, but that is a work around. 

Certainly worth raising a separate issue for this @canweriotnow that we can tag.

**Comment by hallshouse** [comment]
We could pass this on to the CASS project. Yesterday in our initial CASS Project Community meeting, they confirmed that CASS information are meant to be part of statements and stored in the LRS as such. In @garemoko 's example above, a CASS statement can be used for the "accounting professional bodies" group. That statement (according to the CASs project) can have sub-statements assigned to it. In the sub-statements, associations to activities and groups can be defined. With this scenario, the user can remain anonymous as long as they are assigned a group.
